ReactiveCocoa/Swift/SignalProducer.swift

```diff
@@ -581,6 +581,80 @@ public func catch<T, E, F>(handler: E -> SignalProducer<T, F>)(producer: SignalP
 	}
 }
 
+/// Describes how multiple signals or producers should be joined together.
+public enum JoinStrategy: Equatable {
+	/// The signals should be merged, so that any value received on any of the
+	/// input signals will be forwarded immediately to the output signal.
+	///
+	/// The resulting signal will complete only when all inputs have completed.
+	case Merge
```

Comment:
I agree that a separate case seems more appropriate.

Also, I wasn't planning on reimplementing that operator for the first Swift release. :flushed: